docs/preview/02-Features/writing-different-telemetry-types.md

```diff
@@ -429,6 +429,38 @@ logger.LogMetric("Invoice Received", 133.37, telemetryContext);
 
 ## Requests
 
+### Incoming Azure Service Bus requests
+Requests allow you to keep track of Azure Service Bus messages on a queue or topic and how the message was processed.
+
+Here is how you can report an Azure Service Bus queue request on a message that's being processed:
```

Comment:
report or log ?

Reply:
😄 Not sure.

Reply:
I'd say log, but that's a minor thing :)

Reply:
```suggestion
Here is how you can log an Azure Service Bus queue request on a message that's being processed:
```